ui/app/styles/helper-classes/general.scss

```diff
@@ -103,3 +103,6 @@
 .border-radius-2 {
   border-radius: $radius;
 }
+.border-radius-4 {
+  border-radius: 4px;
```

Comment:
Okay also lame, but we have this px already defined as a var. It's used elsewhere which is why I kept it.

Change this to 
```.border-radus-4 {
border-radius: $radius-large
}```